composer.json

```diff
@@ -16,26 +16,23 @@
         "issues": "https://github.com/middlewares/utils/issues"
     },
     "require": {
-        "php": "^7.2 || ^8.0",
-        "psr/http-message": "^1.0",
+        "php": ">=8.1",
```

Comment:
This is a breaking change. 
Can your changes be compatible with php 7.2?

Reply:
Since both PHP 7.2-7.4 and 8.0 are now out of support even for security fixes, I don't have a problem if you want to bump the major version instead.

I would prefer that a new release would support the latest packages for the factories, and not have to deal with unsupported releases to artificially support older PHP versions. If someone want those, they can just use the current version instead :-)

https://www.php.net/supported-versions.php